### PR TITLE
Add option for modal size

### PR DESCRIPTION
**Pull request by Kaylakaze**
Having the modal locked to 2xl is very inconvenient when it only takes a tiny change to make it selectable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This package aims to make it easier to respond with a custom modal when executin
 
 More info for this specific feature can be found in the [Nova Documentation](https://nova.laravel.com/docs/4.0/actions/defining-actions.html#custom-modal-responses).
 
+This is a fork of https://github.com/markwalet/nova-modal-response. The initial purpose of the fork was to add the ability to specify the size of the modal.
+
```
Comment (markwalet) [inline README.md:8]: Can you remove this line again?

**Comment by markwalet** [comment]
Thanks for your contribution. I appreciate that you kept the 2xl size there as a default to prevent breaking changes!

This was very much a tiny package to scratch my own itch. So I'm glas someone else is helping with improving on it. Once you've updated the readme again I'll merge it and tag a new version.

**Review by markwalet — CHANGES_REQUESTED** [review]
Besides the changelog thing, can you minify the asset again?

**Comment by Kaylakaze** [comment]
I'm primarily a backend developer and haven't really used Vue in quite a
few years, so I'm not really sure how to minify the asset these days. I was
mostly providing the PR so you could see the small change. If you want to
throw out my PR, add the change yourself and pack it, I have no objections.

**Comment by markwalet** [comment]
No problem, i want to give you credit to the contribution by  merging this ofcourse. But I can do it myself as well.

I think `npm run prod` should cover the minification. If it doesn't work I can do it myself after Christmas :+1:

**Comment by markwalet** [comment]
Closing this PR because of inactivity. Feature is added in V0.2.1

**Pull request closed without merging**